docs/integrations/destinations/bigquery-migrations.md

```diff
@@ -0,0 +1,12 @@
+# BigQuery Migration Guide
```

Comment:
```suggestion
# BigQuery Upgrade Guide
```

Reply:
This is a bigger change as this header if a metadata validation.  This change will need to be in a separate PR.

Reply:
okay, sounds good. thanks for the context